Source: Upstream
Upstream: commited to 1.0.4 and 1.0.5
Reason: Test fails on 32bit arches
http://pam.cvs.sourceforge.net/viewvc/pam/Linux-PAM/tests/tst-pam_mkargv.c?r1=1.2&r2=1.3
--- tests/tst-pam_mkargv.c	2009/02/25 17:05:45	1.2
+++ tests/tst-pam_mkargv.c	2009/03/03 18:29:04	1.3
@@ -24,8 +24,10 @@
   int myargc;
   char **myargv;
   int argvlen;
+  int explen;
   int i;
 
+  explen = (strlen(argvstring) + 1) * ((sizeof(char)) + sizeof(char *));
   argvlen = _pam_mkargv(argvstring, &myargv, &myargc);
 
 #if 0
@@ -36,7 +38,7 @@
   printf ("\n");
 #endif
 
-  if (argvlen != 333)
+  if (argvlen != explen)
     return 1;
 
   if (myargc != 4)
